### DISCUSSION TITLE
Always have 1 available empty workspace at the end?

### DISCUSSION THREAD

**Discussion by RNGideon** [asmvik/yabai#1813] [Q&A]
Is there a way to accomplish this. I want it so that I always have 1 empty workspace at the end. And then all other spaces are destroyed when they are empty leaving only 1 at the end always available for use.

**Comment by PrayagS** [comment]
https://github.com/koekeishiya/yabai/commit/a51af04ccaac1dbef412d4b1069e49bc9f2df816

Check out the above commit (unreleased). You should be able to listen to these signals and create an empty workspace on `space_created` and then destroy the same on `space_destroyed`.
